db/db_impl/db_impl.cc

```diff
@@ -251,16 +251,18 @@ DBImpl::DBImpl(const DBOptions& options, const std::string& dbname,
   co.num_shard_bits = immutable_db_options_.table_cache_numshardbits;
   co.metadata_charge_policy = kDontChargeCacheMetadata;
   table_cache_ = NewLRUCache(co);
+  SetDbSessionId();
+  std::string db_session_id;
```

Comment:
There should be no need to GetDbSessionId. You can just use `db_session_id_`. Assert not empty if you want.